From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 29 Aug 2022 15:22:10 -0400
Subject: [PATCH] read_password(): only prune CR/NL from the end of the file

Right now, when we read the password/PIN from a file, we're pruning the
end of the string from the file we read indiscriminately.  If you don't
have a newline, that means we're cutting off the final digits of the
text.

This changes it to prune only common special characters from the
pinfile, but also to prune /all/ of them.

Related: rhbz#2122777
Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/password.c | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
